code/datums/wounds/_wounds.dm

```diff
@@ -171,12 +169,32 @@
 
 		victim.visible_message(msg, span_userdanger("Your [limb.plaintext_zone] [occur_text]!"), vision_distance = vis_dist)
 		if(sound_effect)
-			playsound(L.owner, sound_effect, 70 + 20 * severity, TRUE)
+			playsound(L.owner, sound_effect, sound_volume + (20 * severity), TRUE)
```

Comment:
You can get rid of the 20 * severity if this is a var

Reply:
i suppose i could, but then i'd have to go around and change every single sound_volume instance to reflect this

maybe later im a lil low energy tonight